test/test_transforms.py

```diff
@@ -2067,7 +2067,7 @@ def _test_transformation(self, angle, translate, scale, shear, pil_image, input_
                 # https://github.com/python-pillow/Pillow/blob/71f8ec6a0cfc1008076a023c0756542539d057ab/
                 # src/libImaging/Geometry.c#L1060
                 input_pt = np.array([x + 0.5, y + 0.5, 1.0])
-                res = np.floor(np.dot(inv_true_matrix, input_pt)).astype(np.int)
+                res = np.floor(np.dot(inv_true_matrix, input_pt)).astype(np.int32)
```

Comment:
According to the [numpy depreciation notes](https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations), `np.int` is now just `int`? Not sure if it makes a difference in practice in this specific case.

Reply:
Yeah, that is probably better. I didn't know they also accept the builtin types as dtypes.

Reply:
```suggestion
                res = np.floor(np.dot(inv_true_matrix, input_pt)).astype(int)
```